apps/examples/testcase/ta_tc/systemio/utc/utc_spi.c

```diff
@@ -29,7 +29,7 @@
 
 iotbus_spi_context_h spi;
 
-unsigned int bus = 1;
+unsigned int bus = 0;
```

Comment:
Please add static if this variable is used within this file.
Please check another variables also.